web/beacon-app/src/features/projects/routes/ProjectDetailPage.tsx

```diff
@@ -38,7 +38,7 @@ const ProjectDetailPage = () => {
 
   return (
     <AppLayout Breadcrumbs={<ProjectBreadcrumbs project={project} />}>
-      <div className="flex items-center justify-between rounded-md bg-[#F7F9FB] px-6 py-3">
+      <div className="flex items-center justify-between rounded-md bg-[#F7F9FB] px-6 py-4">
```

Comment:
A little padding was added to the top and bottom to for a little space around the gear icon.